File: programacao/c/respostas/selecao/14.c
#include <stdio.h>

int main(void) {
    float p1, p2, p3, media;

    printf("Digite as tres notas: ");
    scanf("%f %f %f", &p1, &p2, &p3);

    media = (p1 * 2 + p2 * 3 + p3 * 5) / 10.0;

    if (media >= 6) {
        printf("Aprovado\n");
    } else if (media > 4) {
        printf("Prova final\n");
    } else {
        printf("Reprovado\n");
    }

    return 0;
}

File: programacao/python/respostas/selecao/14.py
p1 = float(input("Digite a nota da p1: "))
p2 = float(input("Digite a nota da p2: "))
p3 = float(input("Digite a nota da p3: "))

media = (p1 * 2 + p2 * 3 + p3 * 5) / 10.0

if media >= 6:
    print("Aprovado")
elif media > 4:
    print("Prova final")
else:
    print("Reprovado")
